frontend/helpers/comments.ts

```diff
@@ -300,6 +308,30 @@ export const transformCustomsLinksToMarkdown = ( comment: IComment, tickets, typ
 	const usersReferences = text.matchAll(MARKDOWN_USER_REFERENCE_REGEX);
 	const ticketsReferences = text.matchAll(MARKDOWN_TICKET_REFERENCE_REGEX) || [];
 	const resourcesReferences = text.matchAll(MARKDOWN_RESOURCE_REFERENCE_REGEX);
+	const viewpointReferences = text.matchAll(INTERNAL_VIEWPOINT_ID_REGEX);
+
+	if (viewpointReferences) {
+		const teamspace = details.account;
```

Comment:
Destructuring with variable renaming is something that will be useful to you many times

```js
const { account: teamspace, model: projectId, _id: issueId } = details;
```